src/formatDate.js

```diff
@@ -9,6 +9,46 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const splittedDate = date.split(fromFormat[3]);
+  const oldDateObj = {};
+  const result = [];
+
+  for (let i = 0; i < splittedDate.length; i++) {
+    oldDateObj[fromFormat[i]] = splittedDate[i];
+    // make new object via input date;
+  }
+  changeYearsFormat(oldDateObj, toFormat);
+  // this function delete old format for years and make new correct years format
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    result.push(oldDateObj[toFormat[i]]);
+    // make array with new correct date format
+  }
```

Comment:
Similar to the previous suggestion, consider using destructuring for `toFormat` and direct assignment instead of a loop to build the `result` array.